CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 ## 1.11.0
 
+Enhancements:
+
+- Adds `WithAllowsPooling` option
+- Adds `WithURL` option
```

Comment:
Might be nice to have a little blurb about what each of these actually do. 

They don't actually have godocs at the moment, which is also a problem, but probably not one for this change.

Reply:
this would be nice! however, i have no idea what they do!

Reply:
I reached out to the endpoints project to see if anybody had anything additional, but got no response. I may need to nudge them again